website/content/en/preview/getting-started/getting-started-with-eksctl/scripts/setup-roles.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Can we call the file add-roles.sh to be consistent with other files in that dir?